tests/system/general/test_45_FilterSolutions/test_distributed_filter/test_distributed_topology.py

```diff
@@ -0,0 +1,541 @@
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.miniature_hairpin_end_curl_extension = "0 mm"
+        assert info.value.args[0] == "The Tuning Extension property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.MINIATURE_HAIRPIN
+        assert design.topology.miniature_hairpin_end_curl_extension == "0 um"
+        design.topology.miniature_hairpin_end_curl_extension = "2 mm"
+        assert design.topology.miniature_hairpin_end_curl_extension == "2 mm"
+        assert design.topology.circuit_response().splitlines() == read_resource_file(
+            "miniature_hairpin_end_curl_extension.ckt", "Distributed"
+        )
+
+    def test_ring_resonator_end_gap_extension(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.ring_resonator_end_gap_extension = "0 mm"
+        assert info.value.args[0] == "The Tuning Extension property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.RING_RESONATOR
+        assert design.topology.ring_resonator_end_gap_extension == "0 um"
+        design.topology.ring_resonator_end_gap_extension = "2 mm"
+        assert design.topology.ring_resonator_end_gap_extension == "2 mm"
+        assert design.topology.circuit_response().splitlines() == read_resource_file(
+            "ring_resonator_end_gap_extension.ckt", "Distributed"
+        )
+
+    def test_tuning_type_1(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.tuning_type_1 = True
+        assert info.value.args[0] == "The Tuning Type property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.HAIRPIN
+        assert design.topology.tuning_type_1
+        design.topology.tuning_type_1 = False
+        assert design.topology.tuning_type_1 is False
+        assert design.topology.circuit_response().splitlines() == read_resource_file("tuning_type_1.ckt", "Distributed")
+
+    def test_tuning_type_2(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.tuning_type_1 = True
+        assert info.value.args[0] == "The Tuning Type property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.HAIRPIN
+        assert design.topology.tuning_type_1
```

Comment:
Should tuning_type_1 be tuning_type_2? Should it be set to a different value after the default is checked?